azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/DeployMojo.java

```diff
@@ -136,12 +133,17 @@ protected void deployArtifacts(WebAppConfiguration webAppConfiguration)
             final boolean isV1Schema = SchemaVersion
                     .fromString(this.getSchemaVersion()) == SchemaVersion.V1;
             // copy resources according to maven filter if the os is not docker
-            if (webAppConfiguration.getOs() != OperatingSystemEnum.Docker && isV1Schema && this.getDeploymentType() != DeploymentType.JAR) {
-                final List<Resource> effectiveResources = isV1Schema ? resources
+            if (webAppConfiguration.getOs() != OperatingSystemEnum.Docker) {
```

Comment:
For docker, no need to get `getArtifactHandler` and deploy, or we could create a new empty handler for it